caffe-grid/pom.xml

```diff
@@ -42,6 +42,11 @@
   </profiles>
   <dependencies>
     <dependency>
+      <groupId>net.sf.py4j</groupId>
+      <artifactId>py4j</artifactId>
+      <version>0.8.1</version>
+    </dependency>
+    <dependency>
```

Comment:
Any chance to have this dependence only for test only? we want to avoid extra s/w in our jar file.